doc/parameters.md

```diff
@@ -351,3 +351,23 @@ Defines a HTTP proxy to be used for connecting to HTTPS services. Authentication
 **Example:** `foo,bar.com,.extra.dot.com`
 
 Defines a comma-separated list of hostnames and domain names for which the requests should not be proxied. Setting to a single `*` character, which matches all hosts, effectively disables the proxy.
+
+### `APICAST_EXTENDED_METRICS`
+
+**Default:** nil
+**Value:** boolean
+**Example:** "true"
+
+Enables additional information on Prometheus metrics; some labels will be used
+with specific information that will provide more in-depth details about APIcast.
+
+Note: this parameter will create different labels for metrics and can cause
```

Comment:
I think we'll need to measure this to be sure about the performance impact, for example when having 10 or 100 different services.

Maybe we should delete this warning until we measure this and know approximately how many services there needs to be to have a noticeable impact on performance.